CHANGELOG.md

```diff
@@ -21,6 +21,7 @@ _None_
 - Add "Mobile Secrets" to `configure_update` current branch message to clarify which repo it's referring to. [#455]
 - `buildkite_trigger_build` now prints the web URL of the newly scheduled build, to allow you to easily open it via cmd-click. [#460]
 - Add the branch information to the 'This is not a release branch' error that's thrown from complete code freeze lane. [#461]
+- Update `octokit` to `5.6.1` and `danger` to `9.3.0`. These are both major version changes, but as the dependency is internal-only, it shouldn't be a breaking change for clients. [#464]
```

Comment:
Please correct me if I am wrong about this not being a breaking change for clients!

Reply:
Responding here to https://github.com/wordpress-mobile/release-toolkit/pull/464#discussion_r1175610504. 

@AliSoftware The only [breaking change in `5.0.0`](https://github.com/octokit/octokit.rb/releases/tag/v5.0.0) was dropping all versions of Ruby older than 2.7. Since we're already at 2.7.4 everywhere, this shouldn't actually break anything in our client implementations of the Release Toolkit.

Reply:
I moved the `octokit` upgrade to the "Breaking Changes" section: https://github.com/wordpress-mobile/release-toolkit/pull/464/commits/d2a1e86604e42d71d344cb0181c52690e54e827d